### PR TITLE
Setup frontend

### PR DESCRIPTION
**Pull request by ryutaudo**
## What is done in this branch
- Created App component which is a container of this app.
- Added public directory to store bundle and HTML files.
- Created web-pack config file.
- Added scripts of build frontend to package.json

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const defaultState = {
+};
+
+const reducer = (state = defaultState, action) => {
```
Comment (ChristianSchmidt1981) [inline src/reducers/index.js]: please use ES6 here

you can simplify this arrow-function

File: src/containers/App.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { connect } from 'react-redux';
+import App from '../components/App';
+
+
+export default connect()(App);
```
Comment (ChristianSchmidt1981) [inline src/containers/App.js:5]: please add an empty line as last line in every file (eslint)

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+const createStoreWithMiddleware = compose(applyMiddleware(thunk))(createStore);
+const store = createStoreWithMiddleware(reducer);
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  document.getElementById('root'),
+);
```
Comment (ChristianSchmidt1981) [inline src/index.js:18]: please add an empty line as last line in every file (eslint)

File: src/reducers/index.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+  return state;
+};
+
+export default reducer;
```
Comment (ChristianSchmidt1981) [inline src/reducers/index.js]: please add an empty line as last line in every file (eslint)

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  devServer: {
+    contentBase: path.join(__dirname, 'public'),
+    compress: true,
+    port: 8080,
+    proxy: {
+      '/api': 'http://localhost:3000',
+    },
+  },
+};
```
Comment (ChristianSchmidt1981) [inline webpack.config.js:43]: please add an empty line as last line in every file (eslint)

File: src/components/App.jsx [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  render() {
+    return (
+      <div className='app'>
+        Hello World!!
+      </div>
+    );
+  }
+};
+
+export default App;
```
Comment (ChristianSchmidt1981) [inline src/components/App.jsx]: please add an empty line as last line in every file (eslint)

File: package.json [hunk 7]
```diff
@@ -6,6 +6,8 @@
   "author": "Christian Schmidt <cschmidt.solingen@gmail.com>",
   "license": "MIT",
   "scripts": {
+    "build": "webpack",
```
Comment (ChristianSchmidt1981) [inline package.json]: please create a css - file. include it + check if the webpack configuration works with css-files too.
did you want to use plain css, sass or less?

**Review by ChristianSchmidt1981 — CHANGES_REQUESTED** [review]
some smaller issues

File: src/containers/App.js (hunk 2 above)
Comment (ryutaudo) [reply src/containers/App.js:5]: @ChristianSchmidt1981 
It has the empty line but it is not showing up.

File: src/index.js (hunk 3 above)
Comment (ryutaudo) [reply src/index.js:18]: @ChristianSchmidt1981 
It has the empty line but it is not showing up.

File: src/reducers/index.js (hunk 4 above)
Comment (ryutaudo) [reply src/reducers/index.js]: @ChristianSchmidt1981 
It has the empty line but it is not showing up.

File: webpack.config.js (hunk 5 above)
Comment (ryutaudo) [reply webpack.config.js:43]: @ChristianSchmidt1981 
It has the empty line but it is not showing up.

File: src/components/App.jsx (hunk 6 above)
Comment (ryutaudo) [reply src/components/App.jsx]: @ChristianSchmidt1981 
It has the empty line but it is not showing up.

**Review by ChristianSchmidt1981 — APPROVED** [review]
thanks for the feedback.
looks good for me

**Review by eliaahadi — APPROVED** [review]

**Pull request merged**